README.md

```diff
@@ -1,116 +1,48 @@
-
-- Refreshing the NPM package periodically to access new APIs and bug fixes.
-
-- [Using the NPM package according to the instructions](#use-the-npm-package). Do not import the NPM package as commonly done with other NPM packages.
-
-- Do not use the NPM package in an add-in intended for publication to [AppSource](https://appsource.microsoft.com/marketplace/apps?product=office). Add-ins that are published to AppSource must use the Office.js CDN.
-
-- [Using TypeScript definitions for Office.js](#intellisense-definitions).
-
-### Support
-
-The Office.js CDN is the official supported source for Office Add-ins.
-
-For the NPM package sourced through this repository, only the latest version of the package is supported. No support and no patches will be provided for previous versions of the package. The frequency of the updates to this repository and related NPM package to match the CDN version is not guaranteed.
-
-Outlook add-ins don't support hosting Office.js offline due to network access requirements for dependencies like the Microsoft Ajax library.
-
-<br />
-
-## Install the NPM package
-
-To install "office-js" locally via the NPM package, run the following command.
-
-> npm install @microsoft/office-js --save
-
-<br />
-
-Our policy requires that developers always reference the latest version of Office.js library. This is to ensure that essential product updates are served quickly to the Office Add-ins by always referencing the latest release of the library for a given version, such as Generally Available (GA) version. Given that the latest Office.js release is backward-compatible with prior releases, it's safe to update to the most recent release of the library when one is available. Hence, only the latest version of Office.js NPM package is made available for installation.
-
-## Use the NPM package
-
-Installing the NPM package locally creates a set of static Office.js files in the `node_modules\@microsoft\office-js\dist` folder of the directory where you ran the `npm install` command. To use the NPM package, do the following:
-
-1. Either manually or as part of a build script (e.g., `CopyWebpackPlugin` if you're using Webpack) have the files served from a destination of your choosing (e.g., from the `/assets/office-js/` directory of your web server).
-
-2. Reference that location in a `<script>` tag within the HTML file in your add-in project.
-
-For example, if you add the contents of the `dist` folder to the `assets/office-js` directory of your project, then you'd add the following `<script>` tag to your HTML file:
+The Office CDN is the official supported source for Office Add-ins. Reference the Office.js library in the CDN by adding the following `<script>` tag within the `<head>` section of your HTML page.
 
 ```html
-<script src="/assets/office-js/office.js"></script>
+<head>
+    ...
+    <script src="https://appsforoffice.microsoft.com/lib/1/hosted/office.js" type="text/javascript"></script>
+</head>
```

Comment:
Is it worth mentioning that in some cases developers must use special government cloud URLs? Is so, we could just note that there are exceptions and link to our government cloud article.

Reply:
Good call. Added a line about this with a link to our article.